platform/linux-generic/odp_queue_basic.c

```diff
@@ -767,6 +767,98 @@ static void queue_print(odp_queue_t handle)
 	UNLOCK(queue);
 }
 
+#define COL_WIDTH 24
+
+static void queue_print_all(void)
+{
+	uint32_t i, index, len, max_len;
+	const char *name;
+	int status;
+	odp_queue_type_t type;
+	odp_nonblocking_t blocking;
+	odp_queue_op_mode_t enq_mode;
+	odp_queue_op_mode_t deq_mode;
+	odp_queue_order_t order;
+	const char *status_str;
+	const char *bl_str;
+	char type_c, enq_c, deq_c, order_c, sync_c;
+	int prio = 0;
+	odp_schedule_sync_t sync = ODP_SCHED_SYNC_PARALLEL;
+
+	ODP_PRINT("\nList of all queues:\n");
+	ODP_PRINT("------------------\n");
+	ODP_PRINT(" idx %-*s type sta block enq deq ord    len max_len sync prio\n", COL_WIDTH, "name");
```

Comment:
For me the following column naming would be easier to understand without taking more characters:
sta -> stat
block -> blk

Reply:
Fixed in v2